api/server/server.go

```diff
@@ -260,12 +264,74 @@ func NewServer(conf Config) (*Server, error) {
 	}
 
 	s.indexServer = startIndexHTTPServer(s)
+	if err != nil {
```

Comment:
I don't think this is checking an error from starting the index server

Reply:
Uhm, looks like a merge problem. Will fix.